book/types_of_data.md

```diff
@@ -34,89 +32,31 @@ You can cast a string into an Decimal with the `into decimal` command
 
 ## Strings
 
-A string of characters that represents text. There are a few ways we can represent text in Nushell:
-
-**Double quotes**
-
-```
-"my message"
-```
-
-Double quotes are the most common form of quotes and one you may see whenever text is required.
-
-**Single quotes**
-
-```
-'my message'
-```
-
-Single quotes also give you a string value, just like double quotes. The difference here is that they allow you to use double quotes in the text: `'he said "can you grab my glass?"'`
-
-**String interpolation**
-Nushell supports string interpolation, allowing you to run sub-expressions inside of strings prefixed with `$`. For instance:
-
-```
-> echo $"6 x 7 = (6 * 7)"
-6 x 7 = 42
-```
-
-```
-> ls | each { |it| echo $"($it.name) is ($it.size)" }
-───┬─────────────────────
- 0 │ genawait is 4.1 KB
- 1 │ learncpp is 4.1 KB
- 2 │ nuscripts is 4.1 KB
-───┴─────────────────────
-```
-
-**Bare strings**
-
-```
-> echo hello
-```
-
-A unique trait of Nushell is that you can also create a string of one word without any quotes at all.
-
-The above is the same as if we had written:
-
-```
-> echo "hello"
-```
-
-Cast from a string into another type by using the `into <type>` command
-
-```
-> "1" | into int
-> "1.2" | into decimal
-```
-
-Also see [Handling Strings](https://www.nushell.sh/book/loading_data.html#handling-strings).
-
-## Lines
-
-Lines are strings with an implied OS-dependent line ending. When used, the OS-specific line ending is used.
-
-## Column paths
-
-Column paths are a path through the table to a specific sub-table, column, row, or cell.
+A string of characters that represents text. There are a few ways these can be constructed:
 
-Eg) The value `foo.0.bar` in `open data.toml | get foo.0.bar`
+- Double quotes
+  - `"Line1\nLine2\n"`
+- Single quotes
+  `'She said "Nushell is the future".'`
+- Dynamic string interpolation
+  - `$"6 x 7 = (6 * 7)"`
+  - `ls | each { |it| $"($it.name) is ($it.size)" }`
+- Bare strings
+  - `echo hello`
 
-## Glob patterns (wildcards)
-
-In Nushell, file operations may also allow you to pass in a glob pattern, sometimes called a 'wildcard'. This allows you to give a pattern that may match multiple file paths.
-
-The most general pattern is the `*`, which will match all paths. More often, you'll see this pattern used as part of another pattern, for example `*.bak` and `temp*`.
-
-In Nushell, we also support double `*` to talk about traversing deeper paths that are nested inside of other directories. For example, `ls **/*` will list all the non-hidden paths nested under the current directory.
-
-In addition to `*`, there is also the `?` pattern which will match a single character. For example, you can match the word "port" by using the pattern `p???`.
+See [Working with strings](working_with_strings.md) and [Handling Strings](https://www.nushell.sh/book/loading_data.html#handling-strings) for details.
```

Comment:
Globs aren't a data type, so moved to "Moving around your system", which already covers globs (maybe its name could be improved to indicate that it's about working with file/directory paths)